src/test/java/uk/gov/ons/ctp/response/casesvc/message/CaseReceiptReceiverIT.java

```diff
@@ -41,9 +29,18 @@
 @DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
 public class CaseReceiptReceiverIT {
 
-  private UUID collectionExerciseId;
+  private static PubSubEmulator pubSubEmulator;
 
-  private PubSubEmulator pubSubEmulator = new PubSubEmulator();
+  static {
```

Comment:
as this is an instance variable this initialisation should be in a constructor and not a static block